demo/app.js

```diff
@@ -0,0 +1,19 @@
+import { app, h } from '../src/index.js';
```

Comment:
We can safely ignore this, I don't think it should be bundled in hyperapp, this is just for me to demo locally and not have to constantly stash push/pop.